rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -71,7 +76,7 @@ public static void main(String[] args) throws Exception
   @Override
   public int run(String[] args) throws Exception
   {
-    conf = this.getConf();
+    conf = applyRubixSiteConfig(this.getConf());
```

Comment:
lets move this into startServer so that embedded mode can also use it if needed

Reply:
done